scripts/release/debian/test_deb_package.py

```diff
@@ -13,7 +13,7 @@
 
 pytest_base_cmd = '/opt/az/bin/python3 -m pytest -v --forked -p no:warnings --log-level=WARN'
 pytest_parallel_cmd = '{} -n auto'.format(pytest_base_cmd)
-serial_test_modules = ['botservice', 'network', 'cloud', 'appservice', 'iot', 'resource']
+serial_test_modules = ['botservice', 'network', 'cloud', 'appservice', 'iot', 'resource', 'vm']
```

Comment:
`serial_test_modules` is only used to mark these modules' tests as "serial", but being skipped.

Reply:
Pr title updates. 
Concurrent calls in vm test causes this issue in debian